resource-definitions/generic-async-driver/job-pod-template-source-private-repo/agent-runner.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: entity.humanitec.io/v1b1
+kind: Definition
```

Needs `metadata` property:
```suggestion
kind: Definition
metadata:
  id: agent-runner
```